NBitcoin/Crypto/Cryptsharp/SCrypt.cs

```diff
@@ -228,7 +228,8 @@ public static class SCrypt
 			Thread[] threads = new Thread[threadCount - 1];
 			for(int i = 0; i < threads.Length; i++)
 			{
-				(threads[i] = new Thread(workerThread, 8192 * 4)).Start();

```

Comment:
just change `4` to `8`. We are rich

Reply:
Ok but 8192 * 4 != 8192 * 8. 
The PR means the current stack is 32k but when use debugger - 128k.